### PR TITLE
Initial commit for Amazon Web Scraper python script and prerequisites

### PR DESCRIPTION
**Pull request by CaptainRoberts**
# Description

I have an Amazon Web Scraper that gets all the a product has based on the item search URL provided that I wish to share in your repo.

## Type of change

Choosing one or more options from the following as per the nature of your Pull request.
- NOTE: *These boxes can be checked using **`[X]`***

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Documentation Update

# Checklist:
Please tick all the boxes that are fulfilled by your Pull Request.

- [X] I have named my files and folder, according to this project's guidelines.
- [X] My code follows the style guidelines of this project.
- [X] My Pull Request has a descriptive title. (not a vague title like `Update index.md`)
- [X] I have commented on my code, particularly in hard-to-understand areas.
- [X] I have created a helpful and easy to understand `README.md`, according to the given [`README_TEMPLATE.`](https://github.com/Python-World/Python_and_the_Web/blob/master/README_TEMPLATE.md)
- [X] I have included a requirements.txt file (if external libraries are required.)
- [X] My changes do not produce any warnings.
- [X] I have added a working sample/screenshot of the script.

### CODE REVIEW

File: Scripts/Web_Scrappers/Amazon_products_scraper/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+requests
```
Comment (chavarera) [inline Scripts/Web_Scrappers/Amazon_products_scraper/requirements.txt:1]: Add specific version for used module and packages

use freeze list in your virtual environment to create requirements.txt file
```
pip freeze list>requirements.txt
```

**Review by chavarera — CHANGES_REQUESTED** [review]

File: Scripts/Web_Scrappers/Amazon_products_scraper/requirements.txt (hunk 1 above)
Comment (CaptainRoberts) [reply Scripts/Web_Scrappers/Amazon_products_scraper/requirements.txt]: Updated it. Thanks for the suggestion @chavarera

**Review by chavarera — APPROVED** [review]

**Review by AdityaJ7 — APPROVED** [review]
Nice Job Captain

**Comment by AdityaJ7** [comment]
@all-contributors please add @CaptainRoberts for code

**Pull request merged**